### DISCUSSION TITLE
deprecation warning

### DISCUSSION THREAD

**Discussion by gittrolo** [quasarframework/quasar#13335] [General - Components / Directives / etc] [answered]
I use only quasars without any kind of external libraries, how can I fix deprecation warning?
the file package.json does not update automatically?
![image](https://user-images.githubusercontent.com/52664193/166648560-e7c255b7-ec41-4189-b3ea-c2e0d22c6653.png)

**Comment by pdanpdan** [comment, marked as answer]
One or more of:

- check if there is a newer vuex with a fix
- use a lower version of node
- don't use vuex
- ignore warning

**Comment by Hashen110** [comment]
Upgrade vuex version to [v3.6.2](https://github.com/vuejs/vuex/releases/tag/v3.6.2)
